canonical/ubuntu.com
# Set up playwright

## Done
- Set up playwright and write one simple test.

## QA

- Save [this credentials](https://pastebin.canonical.com/p/r3Cf49554s) in `.env.local`
- Command `dotrun`
- Command `yarn playwright test --ui` or `yarn playwright test`
- Check tests pass locally or in github checks

Fix #https://warthogs.atlassian.net/browse/WD-7475
